app/models/actions/propose_assertion.rb

```diff
@@ -26,16 +27,16 @@ def execute
         end
         item.save
       end
-      Event.create(
-        action: 'assertion submitted',
-        originating_user: originating_user,
-        subject: assertion,
-        organization: organization
-      )
+      @subject = assertion
+      create_event('assertion submitted')
       assertion.subscribe_user(originating_user)
       @assertion = assertion
     end
 
+    def state_params
+      nil
```

Comment:
Do assertions not have state params? We do link to them from the events right?

Reply:
The deleted portion that created the Event previously wasn't setting the state params. I'm confused what differentiates the Event's state params from the subject's state params. The subject/assertion themselves does have state params.